application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -376,33 +377,37 @@ record TemporaryData(Instant expiresAt, String duration) {
     /**
      * Gives out advice depending on the {@link ModerationAction} and the parameters passed into it.
      *
-     * @param action gives out a {@link ModerationAction} to be able to respond and build a message
-     *        appropriately such as {@link ModerationAction#MUTE} etc.
-     * @param message is the message that is being used to build responses, such as
-     *        {@link Guild#getName()}, and reasoning for an action.
+     * @param action the action that is being performed, such as banning a user.
+     * @param temporaryData is the message that is being used to display action duration.
```

Comment:
`temporaryData` is not the message. also, u have to explain to the user of the method which temporary data they have to give u.

`@param temporaryData if the action is a temporary action, such as a 1 hour mute`